web/source/keymanweb.js

```diff
@@ -3820,8 +3830,8 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
           if(Pelem.contentDocument.readyState == 'complete') {
             attachFunctor();
           }
-        } else if(keymanweb.setupDesktopElement(Pelem)) {
```

Comment:
We now filter for this earlier, in the main `MutationObserver` method.  No need to re-filter.  `setupDesktopElement` and the old `attachToControl` functionality are also merged now.